lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSignupCard.js

```diff
@@ -66,26 +66,29 @@ class CampaignSignupCard extends React.Component {
   }
 
   render() {
+    const header = this.renderHeader();
     return (
-      <article className="beta-card">
-        <div className="beta-card__body">
+      <li>
```

Comment:
Can you follow sort of what I did for the slide that renders ReportbackItems, with not having it output the `<li>`s in the component. For the slide with the single campaign card, this will output an `<li>` tag which we don't want.